__tests__/commands/run.js

```diff
@@ -133,3 +144,20 @@ test('adds quotes if args have spaces and quotes', (): Promise<void> => {
     expect(execCommand).toBeCalledWith(...args);
   });
 });
+
+test('adds workspace root node_modules/.bin to path when in a workspace', (): Promise<void> => {
+  return runRunInWorkspacePackage('packages/pkg1', ['env'], {}, 'workspace', (config, reporter): ?Promise<void> => {
+    const logEntry = reporter.getBuffer().find(entry => entry.type === 'log');
+    const logData = logEntry ? logEntry.data.toString() : '{}';
+    const parsed = logEntry ? JSON.parse(logData) : {};
+    let envPaths = [];
+    if (parsed.PATH) {
+      envPaths = parsed.PATH.split(':');
+    } else if (parsed.Path) {
+      envPaths = parsed.Path.split(';');
+    }
+
+    expect(envPaths).toContain(path.join(config.cwd, 'node_modules/.bin'));
```

Comment:
I expected this to fail on Windows since the path separator should be `\` there. Weird.

Reply:
IIRC `path.join()` converts it auto-magically.

Reply:
although... why didn't I do `path.join(config.cwd, 'node_modules', '.bin')`? 😕 ...I'm an idiot 😆

Reply:
Wow, nice.